### PR TITLE
Debug xen problems

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

File: v2v/tests/src/function_test_xen.py [hunk 1]
```diff
@@ -224,10 +223,12 @@ def check_result(result, status_error):
         bk_xml = None
         os.environ['LIBGUESTFS_BACKEND'] = 'direct'
 
+        # See man virt-v2v-input-xen(1)
+        process.run('update-crypto-policies --set LEGACY')
```
Comment (chunfuwen) [inline v2v/tests/src/function_test_xen.py]: add shell=true parameter when call process.run()

**Comment by xiaodwan** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/2403

**Comment by xiaodwan** [comment]
https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/v2v/view/RHEL-8.1/job/v2v-RHEL-8.1-runtest-x86_64-function-function_test_xen-rhel/28/testReport/rhel/function_test_xen/

The xen job was not blocked.

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_xen.positive_test.libvirt.pool_uuid.output_mode.libvirt: PASS (637.72 s)
(1/1) type_specific.io-github-autotest-libvirt.function_test_xen.positive_test.libvirt.display.vnc.encrypt.output_mode.libvirt: PASS (669.97 s)

File: v2v/tests/src/function_test_xen.py (hunk 1 above)
Comment (xiaodwan) [reply v2v/tests/src/function_test_xen.py]: Done

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh Please help to review this PR, thanks.

**Pull request merged**